### ISSUE TITLE
support `tip` 

### ISSUE THREAD

**Issue by staabm** [spaze/phpstan-disallowed-calls#125]
phpstan [RuleErrorBuilder](https://github.com/phpstan/phpstan-src/blob/1c012538fee1865bf3cd2b756085a462b3840b6a/src/Rules/RuleErrorBuilder.php#L104-L110) supports a `tip` which brings additional information besides the error message.

this tip is rendered below a error message in the analysis result

would be great a `tip` could be configured in the config file, like

```
    disallowedStaticCalls:
        -
            method: 'PotentiallyDangerous\Debugger::log()'
            message: 'use our own logger instead'
            tip: 'see https://our-docs.com/logging on how logging should be used'
```

**Comment by spaze** [comment]
Sounds like a good idea, thanks for the suggestion! I've added it in #128 and released [2.7.0](https://github.com/spaze/phpstan-disallowed-calls/releases/tag/v2.7.0) a few moments ago. Note that the config key is called `errorTip` to follow `errorIdentifier`, not just `tip`.

**Comment by staabm** [comment]
thanks for the fast feedback and impl.
